application-development/helloworld-springboot-auth/pom.xml

```diff
@@ -0,0 +1,85 @@
+<!--
+~ Copyright (c) 2017 Google Inc. All Rights Reserved.
```

Comment:
You can remove the 

"All Rights Reserved."

and just use "Copyright (c) 2017 Google Inc."